<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Đăng Ký</string>
  </property>
  <widget class="QLabel" name="role_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>223</y>
     <width>278</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Phân quyền:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>128</y>
     <width>278</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="username_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>278</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tài khoản:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>194</y>
     <width>278</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="role_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>261</y>
     <width>278</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Quản lý</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nhân viên</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>157</y>
     <width>278</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Mật khẩu:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="register_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>278</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Đăng Ký</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>278</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Họ và tên</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hovaten_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>278</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
